mopidy_local/www/index.html

```diff
@@ -1,43 +1,52 @@
 <!doctype html>
 <html>
   <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Mopidy-Local</title>
+    <link rel="stylesheet" href="/mopidy/mopidy.css" type="text/css" media="screen, print">
     <style>
-      html {
-          background: #f8f8f8;
-          color: #555;
-          font-family: Geneva, Tahoma, Verdana, sans-serif;
-          line-height: 1.4em;
+      #thumbnail_container {
+       display: grid;
+       grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+       grid-gap: 2px;
+       line-height: 0%;
       }
-      body {
-          margin: 0 auto;
-          max-width: 600px;
+      #thumbnail_container a {
+       border: none;
       }
-      h1 {
-          font-weight: 500;
-          line-height: 1.1em;
-      }
-      a {
-          color: #555;
-          text-decoration: none;
-      }
-      img {
-          border: 0;
-          height: 80px;
-          width: 80px;
+      #thumbnail_container img {
+       border: none;
+       width: 100%;
       }
     </style>
   </head>
   <body>
-    <h1>Mopidy-Local</h1>
-    <p>
-      This Web client is used to serve album art extracted from local
-      media files by the Mopidy-Local extension.
-    </p>
-    <div>
-      {% for image in images %}
-        <a href="{{image}}"><img src="{{image}}"></a>
-      {% end %}
+    <div class="box focus">
+      <h1>Mopidy-Local</h1>
+      <p>
+        This Web client is used to serve album art extracted from local
+        media files by the Mopidy-Local extension.
+      </p>
+    </div>
+    <div class="box">
+      <h2>Mopidy-Local image cache</h2>
+      <span id="thumbnail_container">
+        {% for image in images %}
+        <a href="{{image}}">
+          <img id="thumbnail" src="{{image}}" alt="Album art thumbnail; link to full size album art" loading="lazy">
+        </a>
+        {% end %}
+      </span>
     </div>
   </body>
 </html>
+<script>
+  var imgwidth = document.getElementById('thumbnail').width;
+  var css = '#thumbnail_container img {height: ' + imgwidth + 'px;}';
+  var head = document.head || document.getElementsByTagName('head')[0];
+  var style = document.createElement('style');
+  head.appendChild(style);
+  style.type = 'text/css';
+  style.appendChild(document.createTextNode(css));
+</script>
```

Comment:
This needs to be moved to just before the `</body>` tag.

Is this necessary though? I would make the following changes:

- Change `<span id="thumbnail_container">` to a `<div>`
- Add `display: block` and `height: auto` to `#thumbnail`

I think that will achieve the desired affect. I've been out of the webdev industry for a year now so my CSS is definitely getting a bit rusty, but I'm reasonably certain you can achieve the desired outcome without Javascript.